cfgov/jinja2/v1/events/_macros.html

```diff
@@ -130,8 +130,20 @@
 {% macro event_venue(event, event_state) %}
     {%- set city =  event.venue_city + ', ' + event.venue_state | default('Washington, DC') -%}
     <section class="event-venue">
-        <header class="event-venue_header">
-            <h2 class="event-venue_heading">{{ city }}</h2>
+        <div class="event-venue_details">
+            <header class="event-venue_header">
+                {% if ( event_state == 'present' ) %}
+                <a class="btn
+                          event-venue_live-btn"
+                   href="/">
```

Comment:
Converted to a button in https://github.com/cfpb/cfgov-refresh/pull/1625/commits/36f6195becc4c28009ba7b84c31290ffe07277ae.